dot/telemetry/mailer_test.go

```diff
@@ -159,7 +174,8 @@ func TestListenerConcurrency(t *testing.T) {
 
 	defer cancel()
 
-	resultCh = make(chan []byte)
+	resultCh := make(chan []byte)
+	mailer := bootstrapMailer2Test(t, resultCh)
```

Comment:
Is resultsCh used at all? :thinking:

Reply:
Added an assertion to the `resultCh` messages